assets/javascripts/discourse/connectors/user-card-metadata/gamification-score.hbs

```diff
@@ -1,6 +1 @@
-{{#if user.gamification_score}}
-  <h3>
```

Comment:
@chapoi this PR would change the H3 in the user card to a <dl> <dt> is that going to cause problems? Things looks good in the screenshots provided from my point of view, but I wanted to get your advice.

Reply:
I would prefer to keep the h3 tag for visual hierarchy and screenreaders. The <dd> tag also needs to be kept otherwise the whole syntax makes little sense.

Reply:
Okay I brought back the <h3> tag for the user card score:
<img width="1221" alt="Screenshot 2023-06-01 at 10 47 02 AM" src="https://github.com/discourse/discourse-gamification/assets/70915823/fd35a19b-7362-4af9-8b89-e4268cc80388">

And <dd> tag for the user profile score: 
<img width="1221" alt="Screenshot 2023-06-01 at 10 47 38 AM" src="https://github.com/discourse/discourse-gamification/assets/70915823/63377760-c459-4fe8-a57b-e4c0d21a7e7a">

The color for the user card score is gray and for the user profile score is blue. Is that going to be an issue?

Reply:
Okay I brought back the <h3> h3 tag for the user card score:
<img width="1221" alt="Screenshot 2023-06-01 at 10 47 02 AM" src="https://github.com/discourse/discourse-gamification/assets/70915823/fd35a19b-7362-4af9-8b89-e4268cc80388">

And <h3> dd tag for the user profile score: 
<img width="1221" alt="Screenshot 2023-06-01 at 10 47 38 AM" src="https://github.com/discourse/discourse-gamification/assets/70915823/63377760-c459-4fe8-a57b-e4c0d21a7e7a">

The color for the user card score is gray and for the user profile score is blue. Is that going to be an issue?

Reply:
I think it would be best if it's blue in both places, that probably makes it more clear the usercard one is clickable too. After that you can approve on my end @janzenisaac